src/02-common-principles.md

```diff
@@ -191,23 +191,30 @@ apply to different runs and rec files. Also if the JSON file
 (`task-xyz_acq-test1_bold.json`) is defined at dataset top level directory, it
 will be applicable to all task runs with `test1` acquisition parameter.
 
-Example 3: Multiple json files at different levels for same task and acquisition
-parameters
+Example 3: Multiple json files at different levels for same task and
+acquisition parameters
 
 ```Text
-sub-01/
-   sub-01_task-xyz_acq-test1_bold.json
-         anat/
-         func/
-             sub-01_task-xyz_acq-test1_run-1_bold.nii.gz
-             sub-01_task-xyz_acq-test1_rec-recon1_bold.nii.gz
-             sub-01_task-xyz_acq-test1_rec-recon2_bold.nii.gz
+task-xyz_acq-test1_bold.json
+    sub-01/
+        anat/
+        func/
+            sub-01_task-xyz_acq-test1_run-1_bold.nii.gz
+            sub-01_task-xyz_acq-test1_rec-recon1_bold.nii.gz
+            sub-01_task-xyz_acq-test1_rec-recon2_bold.nii.gz
+            sub-01_task-xyz_acq-test1_bold.json
 ```
 
-In the above example, the fields from `task-xyz_acq-test1_bold.json` file will
-apply to all bold runs. However, if there is a key with different value in
-`sub-01/func/sub-01_task-xyz_acq-test1_run-1_bold.json`, the new value will be
-applicable for that particular run/task NIfTI file/s.
+In the above example, the fields from the `task-xyz_acq-test1_bold.json` file
+at the top directory will apply to all bold runs. However, if there is a key
+with different value in the
+`sub-01/func/sub-01_task-xyz_acq-test1_run-1_bold.json` file defined at a
+lower level, that value will be applicable for that particular run/task NIfTI
+file/s. In other words, the `json` file at the lower level overrides values
```

Comment:
Do we use "low" and "high" consistently? I often think of the lower level as closer to the root. It might be less ambiguous to use "shallow" and "deep".

Reply:
good point. shallow and deep is much better. :+1: